<!-- -*- mode: XML -*- -->

<launch>

  <node pkg="tf2_ros" type="static_transform_publisher" name="tf_map"
              args="327999 4691984 0 0 0 0 /earth /loacal_map" />

  <include file="$(find osm_cartography)/launch/viz_osm.launch">
    <arg name="url" value="package://pev_odom/osm/media_lab.osm" />
  </include>

  <node pkg="pev_map_server" type="pev_map_server" name="pev_map_server_node" output="screen">
    <param name="table_name" type="str" value="map" />
    <param name="file_path" type="str" value="$(find pev_map_server)" />
  </node>

  <!-- Map server -->
  <!-- <arg name="map_file" default="$(find pev_odom)/maps/media_lab_yard.yaml"/>
  <node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file)" /> -->

  <!-- Send a static GPS fix to every new subscriber. Edit latitude and longitude in launch/demo.gps to use your own position. -->
  <node pkg="rostopic" type="rostopic" name="fake_gps_fix"
        args="pub /fake_gps/fix sensor_msgs/NavSatFix
              --latch
              --file=$(find pev_sensors)/config/yard.gps"
        output="screen"/>

  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find pev_map_server)/rviz/map_server.rviz" />

</launch>
